media/javascript/readthedocs-analytics.js

```diff
@@ -2,33 +2,40 @@
 // https://docs.readthedocs.io/en/latest/advertising-details.html#analytics
 
 
-// RTD Analytics Code
+// Skip analytics for users with Do Not Track enabled
+if (navigator.doNotTrack === '1') {
```

Comment:
Should the JS respect the setting as well? Probably needs to be added to the context manager.

Reply:
I don't understand this comment. Can you elaborate?

Reply:
We shouldn't add this JS opt out unless the DO_NOT_TRACK_ENABLED setting is True

Reply:
Ideally, yes we want to only respect DNT if people running RTD have the setting enabled. However, the use case we are supporting by changing that is to help people taking the readthedocs.org code base and don't want to support DNT.

To do this, we would need to pass the DNT setting through the `READTHEDOCS_DATA` object. We *can* do that, but I don't think it's worth it. Do you think it is?

Reply:
Yea, that makes sense for user docs. I guess I was thinking for the `base.html` more than the `analytics.js`.

Reply:
I think that's the only tidbit I'd add before it's deployable -- just so that all our users dashboards don't get DNT'd automatically.

Reply:
I'm going to add this to `base.html`, but user dashboards are going to get DNT'd. The only thing this will change is that people who take the readthedocs.org codebase and use it in their infrastructure won't get DNT'd automatically on their installation except on docs pages where they still will.